Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string userFirstName = GetStringInput(message: "Enter your first name: ");
+            string userSurname = GetStringInput(message: "Enter your surname: ");
+            int userAge = GetIntInput(message: "Enter your age: ");
+            float userHeight = GetFloatInput(message: "Enter your height in cm: ");
+            float userWeight = GetFloatInput(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
```

Comment:
This has to run twice. That won't change your test results, so don't worry about that.

Reply:
It's okay, spec was poor